src/App/DeliveryOptions/Service/DeliveryOptionsService.php

```diff
@@ -140,7 +142,9 @@ private function createCarrierSettings(Carrier $carrier, PdkCart $cart): array
             ->pluck('weekday')
             ->toArray();
 
-        $minimumDropOffDelay = $cart->shippingMethod->minimumDropOffDelay;
+        $minimumDropOffDelay = -1 === $cart->shippingMethod->minimumDropOffDelay
```

Comment:
Dit lijkt een bugfix die in een aparte PR hoort? Negeer dit als ik het verkeerd heb.